scripts/chains/smingot_chain.zs

```diff
@@ -0,0 +1,328 @@
+import crafttweaker.item.IIngredient;
+
+val craft_removals as string[] = 
+[
+	"nuclearcraft:nuclearcraft:smore:0",
+	"nuclearcraft:nuclearcraft:moresmore:0",
+	"nuclearcraft:nuclearcraft:foursmore:0",
+];
+for craft_name in craft_removals {
+	recipes.removeByRecipeName(craft_name);
+}
+
+assembler.recipeBuilder()
+	.inputs(<ore:ingotMarshmallow>)
+	.inputs(<nuclearcraft:graham_cracker> * 2)
+	.inputs(<ore:ingotHotMilkChocolate>)
+	.outputs(<nuclearcraft:smore>)
+	.EUt(1920)
+	.duration(100)
+	.buildAndRegister();
+	
+assembler.recipeBuilder()
+	.inputs(<ore:ingotMarshmallow>)
+	.inputs(<nuclearcraft:smore> * 2)
+	.inputs(<ore:ingotHotMilkChocolate>)
+	.outputs(<nuclearcraft:moresmore>)
+	.EUt(7680)
+	.duration(200)
+	.buildAndRegister();
+	
+assembler.recipeBuilder()
+	.inputs(<ore:ingotMarshmallow>)
+	.inputs(<nuclearcraft:moresmore> * 2)
+	.inputs(<ore:ingotHotMilkChocolate>)
+	.outputs(<nuclearcraft:foursmore>)
+	.EUt(30720)
+	.duration(400)
+	.buildAndRegister();
```

Comment:
I'd like to suggest the forming press for the smores instead. You do press them together IRL after all.